frappe/database/query.py

```diff
@@ -283,6 +284,36 @@ def add_conditions(self, conditions: frappe.qb, **kwargs):
 
 		return conditions
 
+	@staticmethod
+	def get_condition_from_nested_sets(value: list | tuple, table: str):
+		field = frappe.meta.get_field("name")
```

Comment:
didn't understand this 🤔

Reply:
https://github.com/frappe/frappe/blob/develop/frappe/model/db_query.py#L583
moving towards integer primary keys 😆

Reply:
removed this 👍🏿